### PR TITLE
Copy button

### PR DESCRIPTION
**Pull request by simonseyock**
### Description:
This PR adds a dedicated CopyButton that copies features on click. Also this PR addresses an issue with the SelectFeaturesButton which was clearing the feature collection on rerender. This is originally caused by the ToggleButton which seems to call `onToggle` on rerender. This change makes the button more robust regardless.

### CODE REVIEW

File: src/Button/CopyButton/CopyButton.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
```
Comment (dnlkoch) [inline src/Button/CopyButton/CopyButton.tsx:2]: The `react` imports can be combined.

File: src/Button/CopyButton/CopyButton.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import OlGeometry from 'ol/geom/Geometry';
+import OlVectorLayer from 'ol/layer/Vector';
+
+import AnimateUtil from '@terrestris/ol-util/dist/AnimateUtil/AnimateUtil';
+
+import { CSS_PREFIX } from '../../constants';
+import { useMap } from '../../Hook/useMap';
+import { DigitizeUtil } from '../../Util/DigitizeUtil';
+import SelectFeaturesButton, { SelectFeaturesButtonProps } from '../SelectFeaturesButton/SelectFeaturesButton';
+import { SelectEvent } from 'ol/interaction/Select';
```
Comment (dnlkoch) [inline src/Button/CopyButton/CopyButton.tsx:14]: Minor: import can be moved to other `ol` imports further up.

File: src/Button/CopyButton/CopyButton.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+export type CopyButtonProps = OwnProps & Omit<SelectFeaturesButtonProps, 'layers'>;
+
+/**
+ * The className added to this component.
+ */
+const defaultClassName = `${CSS_PREFIX}copybutton`;
+
+/**
+ * The CopyButton.
+ */
```
Comment (dnlkoch) [inline src/Button/CopyButton/CopyButton.tsx:33]: In my view this comment is not needed and can be removed.

**Review by dnlkoch — APPROVED** [review]
Thanks @simonseyock!

File: src/Button/CopyButton/CopyButton.tsx (hunk 1 above)
Comment (simonseyock) [reply src/Button/CopyButton/CopyButton.tsx:2]: i think the `import *` cannot be combined.

Comment (dnlkoch) [reply src/Button/CopyButton/CopyButton.tsx]: See 323de2fd2b95c17038b5c0b364acbf407769c204.

File: src/Button/CopyButton/CopyButton.tsx (hunk 2 above)
Comment (dnlkoch) [reply src/Button/CopyButton/CopyButton.tsx]: See 248abbeb70b02f23cf704dc867893ae3a5bd9d77.

**Comment by dnlkoch** [comment]
@simonseyock I added some minor code-style commits. Feel free to merge!

**Pull request merged**